File: backend/src/main/java/org/jdk/project/dto/digitaltwin/DigitalTwinRobotDto.java
package org.jdk.project.dto.digitaltwin;

import lombok.Builder;
import lombok.Getter;

@Getter
@Builder
public class DigitalTwinRobotDto {
  private Long id;
  private String robotId;
  private String robotName;
  private Long regionId;
  private String regionName;
  private String roomCode;
  private String coordinate;
  private String workingState;
  private Integer batteryState;
}

File: backend/src/main/kotlin/org/jdk/project/dto/digitaltwin/DigitalTwinRobotDto.kt
package org.jdk.project.dto.digitaltwin

data class DigitalTwinRobotDto(
    var id: Long? = null,
    var robotId: String? = null,
    var robotName: String? = null,
    var regionId: Long? = null,
    var regionName: String? = null,
    var roomCode: String? = null,
    var coordinate: String? = null,
    var workingState: String? = null,
    var batteryState: Int? = null,
)
